Tests/IGListCollectionViewTests.m

```diff
@@ -0,0 +1,65 @@
+//
+//  IGListCollectionViewTests.m
+//  IGListKit
+//
+//  Created by Jeff Bailey on 12/7/16.
+//  Copyright © 2016 Instagram. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import <IGListKit/IGListKit.h>
+
+static const CGRect kIGListCollectionViewTestFrame = (CGRect){{0.0, 0.0}, {100.0, 100.0}};
+
+@interface IGListCollectionViewTests : XCTestCase
+
+@property (nonatomic, strong) UIWindow *window;
```

Comment:
Yes, the window is needed because of when UIAppearance properties are applied.  It fails without it.  It was not what I was expecting and it took me a while to figure this out.